### ISSUE TITLE
Terraform bootstrap failed and admin cannot log in for new Rancher install

### ISSUE THREAD

**Issue by dnoland1** [rancher/rancher#31398]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
Unknown how to reproduce. Installed new Rancher environment using Terraform which has been used many times in the past. This Rancher environment is a downstream cluster managed by Rancher.

**Result:**
First, the Terraform bootstrap failed with this error:
```
Error: [ERROR] Connecting with user/pass: Bad response statusCode [403]. Status [403 Forbidden]. Body: [message=settings.management.cattle.io "k8s-version" is forbidden: User "user-z86z5" cannot get resource "settings" in API group "management.cattle.io" at the cluster scope, baseType=error, code=Forbidden] from [https://noland-industries.saas-dev.rancher.space/v3/settings/k8s-version]
```
Was able to go to the URL and enter a new password, but after that, could not log in as admin. No error on login screen.

Rancher logs are attached:
[rancher-c8459fbdd-8k9m2-previous-2021-02-19_1343.log](https://github.com/rancher/rancher/files/6013053/rancher-c8459fbdd-8k9m2-previous-2021-02-19_1343.log)
[rancher-c8459fbdd-8k9m2-2021-02-19_1343.log](https://github.com/rancher/rancher/files/6013054/rancher-c8459fbdd-8k9m2-2021-02-19_1343.log)
[rancher-c8459fbdd-xjhrh-2021-02-19_1343.log](https://github.com/rancher/rancher/files/6013055/rancher-c8459fbdd-xjhrh-2021-02-19_1343.log)

No projects exist in the environment:
```
$ kubectl get projects.management.cattle.io -A --show-labels
No resources found
```

YAML for the cluster:
[c-rglbf.yaml.txt](https://github.com/rancher/rancher/files/6013046/c-rglbf.yaml.txt)

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
v2.5.5
- Installation option (single install/HA):
HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
m5a.large on AWS
- Kubernetes version (use `kubectl version`):

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-14T05:14:17Z", GoVersion:"go1.15.6", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.7+k3s1", GitCommit:"5a00e38db4c198fb0725a6b709380aed8053d637", GitTreeState:"clean", BuildDate:"2021-01-14T23:09:21Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):
container

gz#15258

**Comment by Oats87** [comment]
This appears to have happened because the corresponding global role binding for the admin user was not created. Manually creating a global role binding for the `user-XXXXX` to the global role `admin` fixed the issue and allowed login.

**Comment by dnoland1** [comment]
looks like when rancher first started and was bootstrapping, pod restarted and didn't resume where is left off, leaving the environment in a bad state.

**Comment by rawmind0** [comment]
@dnoland1 , PR https://github.com/rancher/terraform-provider-rancher2/pull/679 should address this issue, waiting for `local` cluster is active before start the bootstrap. This is included at Rancher2 tfp v1.16.0. 

Please reopen the issue if required.
